### ISSUE TITLE
Issues with click and drag

### ISSUE THREAD

**Issue by alanorth** [robotrovsky/Linux-Magic-Trackpad-2-Driver#23]
This is a bit hard to explain, but I am having issues with physical click and drag using one finger. This causes problems with, for example, selecting files in the file manager, moving windows, selecting text on a webpage, etc. holding with thumb and dragging, as well as double tap and drag work fine. I'm using the latest version of the driver on Linux 4.18.

Here is a video demonstrating the problem: https://youtu.be/L-U8M6quwXA

**Comment by mwyborski** [comment]
@alanorth Thanks for reporting back! To be honest i haven't had the time to test the patch in Linux 4.18. But maybe this is a configuration issue, when i get back to my trackpad in 2 weeks after vacation i will test it. But i think i would have noticed this. Which Desktop and driver backend (xf86-mtrack or xf86-libinput) are you using? Maybe there is someone who in the meantime could check this...

**Comment by bobbysue** [comment]
I haven't been able to reproduce the bug.  I've also been testing on kernel v4.18, but using chromiumOS (so no xf86).

**Comment by alanorth** [comment]
@robotrovsky you're right, I forgot to give those critical details! This behavior happens for me on Xorg and Wayland, using libinput driver on both. Current versions on Arch Linux:

- libinput 1.11.3-1
- xf86-input-libinput 0.28.0-1 
- xorg-server 1.20.1-1
- wayland 1.16.0-1

**Comment by adam-h** [comment]
I've not tried libinput recently, but xf86-input-mtrack is working fine for click and drag on Arch.

**Comment by alanorth** [comment]
I'm trying to keep my setup in Wayland so I can't test `xf86-input-mtrack`. I have no other issues with this driver, and my workaround is either to double tap and then select/drag, or use my thumb and select/drag. Would you like me to close this issue, @robotrovsky?

**Comment by yiding** [comment]
I also ran into this issue. I think it's something to do with https://github.com/robotrovsky/Linux-Magic-Trackpad-2-Driver/commit/8d93f27b96b19373c70e93af8d39ab58aec7ecd5. Back when I was on kernel 4.17 I just used the rev previous to this one to avoid the issue.

I'm on fedora 28, happens with both wayland and x11.
xorg-x11-libinput 0.27.1.2
xorg-x11-server-Xorg 1.18.6-8
libinput 1.11.3-1
kernel 4.18.8-200
wayland 1.15.0
